no clear instructions on how to build on linux

I made a git clone of the project but I am unable to compile.

This is what I did:

~/Work/OpenWebStart$ ./mvnw clean install

but I get this error (some parts were removed):
....
[INFO] Building OpenWebStart 1.2.0-SNAPSHOT
[INFO] ------------------------------------------------------------------------
[WARNING] The POM for net.adoptopenjdk:icedtea-web-all-dependencies:jar:2.0.0-SNAPSHOT is missing, no dependency information available
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO] 
[INFO] OpenWebStart Parent ................................ SUCCESS [  3.647 s]
[INFO] Resource Transformer for ITW ....................... SUCCESS [  4.329 s]
[INFO] OpenWebStart ....................................... FAILURE [  0.348 s]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
....
[ERROR] Failed to execute goal on project openwebstart: Could not resolve dependencies for project com.openwebstart:openwebstart:jar:1.2.0-SNAPSHOT: Failure to find net.adoptopenjdk:icedtea-web-all-dependencies:jar:2.0.0-SNAPSHOT in https://jcenter.bintray.com/ was cached in the local repository, resolution will not be reattempted until the update interval of jcenter has elapsed or updates are forced -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <goals> -rf :openwebstart



~/Work/OpenWebStart$ lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 18.04.3 LTS
Release:	18.04
Codename:	bionic

~/Work/OpenWebStart$ javac -version && java -version
javac 11.0.4
openjdk version "11.0.4" 2019-07-16
OpenJDK Runtime Environment (build 11.0.4+11-post-Ubuntu-1ubuntu218.04.3)
OpenJDK 64-Bit Server VM (build 11.0.4+11-post-Ubuntu-1ubuntu218.04.3, mixed mode, sharing)
~/Work/OpenWebStart$ mvn --version
Apache Maven 3.6.0
Maven home: /usr/share/maven
Java version: 11.0.4, vendor: Ubuntu, runtime: /usr/lib/jvm/java-11-openjdk-amd64
Default locale: en_US, platform encoding: UTF-8
OS name: "linux", version: "5.0.0-36-generic", arch: "amd64", family: "unix"


I can create a INSTALL file with what is needed

--- a/INSTALL.md
+++ b/INSTALL.md
@@ -1,0 +1,17 @@
+# Ubuntu Installation
+This was tested on Ubuntu 18.04.3 LTS but should be similar on other linux distributions.
+
+Please mind that at the time of the writing (Dec 2019) the compilation is possible only with Java 1.8.
+
+Here are the steps needed to be done:
+
+1. Install the prerequisite tools: sudo apt install git maven openjdk-8-jdk
+2. Clone this repository
+3. Clone the [IcedTea-Web](https://github.com/AdoptOpenJDK/IcedTea-Web) project.
+4. `export JAVA_HOME=/usr/lib/jvm/java-8-openjdk-amd64/` - in case you have multiple java installations it is safer
+5. Go to the IcedTea-Web clone directory and type: mvn install
+6. Go back to this clone and type: ./mvnw install
+
+
+
+

